scripts/generate_js_definitions.sh

```diff
@@ -41,6 +41,7 @@ else
     npm install # in case things have changed
     npm run fetch:local
     npm run build
+    npm run build:types-json
```

Comment:
This seems redundant since `build:types-json` is now part of the `postbuild` script

Reply:
I'm not seeing where `postbuild` gets called, do you know what that script is for?

Reply:
The way npm scripts work, any script that starts with "pre" or "post" runs before/after the main script of the same name. So "postbuild" automatically runs after "build".